tiledb/type/range/test/unit_check_range_is_subset.cc

```diff
@@ -131,7 +131,7 @@ TEMPLATE_TEST_CASE(
 
 TEMPLATE_TEST_CASE(
     "Test check_is_subset for floating-point types", "[range]", float, double) {
-  TestType superset_data[2]{-10.5, 3.33};
+  TestType superset_data[2]{-10.5, 3.33f};
```

Comment:
Any idea why the annotation is only required on the second item?

Reply:
I assume it's because it's -actually looking at the specific values- and finding that there will be bit truncation of the not exactly representable x.33 double numbers when assigning those double values to a float (non-zero bits in mantissa of double that will be lost when assigning to the float).
(https://stackoverflow.com/questions/52095592/c-warning-initializing-truncation-from-double-to-float-in-one-declaration-but)